src/Http/Controllers/Auth/RegisterController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers\Auth;
+
+use Blumilk\Meetup\Core\Http\Controllers\Controller;
+use Blumilk\Meetup\Core\Http\Requests\RegisterUserRequest;
+use Blumilk\Meetup\Core\Models\User;
+use Illuminate\Contracts\Hashing\Hasher;
+use Illuminate\Contracts\View\View;
+
+class RegisterController extends Controller
+{
+    public function create(): View
+    {
+        return view("user.register");
+    }
+
+    public function store(RegisterUserRequest $request, Hasher $hasher): View
+    {
+        $user = User::create([
+            "name" => $request->get("name"),
+            "email" => $request->get("email"),
+            "password" => $hasher->make($request->get("password")),
```

Comment:
and move passwor hasher to model i mean .every time you create a model, immediately hash the changes to password

Reply:
do we inject dependency to model or use facades?

Reply:
@MrHDOLEK, what do you mean here?

Reply:
@krzysztofrewak I mean to set up password hashing when creating the model because actual should be use `$hasher->make`

Reply:
Then model itself should be free of dependencies like Hasher, I think. It could be done in creating() method of user observer.